### PR TITLE
feat: use publicRuntimeConfig for toggles configuration

### PR DESCRIPTION
**Pull request by superbiche**
Simple PR to allow configuration in `publicRuntimeConfig.featureToggle` in `nuxt.config.js`.  
Runtime config will override plugin configuration in `options.featureToggle` as commonly seen in other modules, I chose to make  `moduleOptions` overriden by runtime config as well. Let me know if you think it's OK.

(also bumped a few deps as dependabot's PRs are stale).

### CODE REVIEW

**Review by stephenkr — APPROVED** [review]
Hi @superbiche this looks great, thank you for contributing.
Would you also be able to update the Readme.md with some documentation on using this module with `publicRuntimeConfig`

Thank you again.

File: examples/demo/package.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
   "dependencies": {
     "@nuxtjs/pwa": "^3.0.0-beta.20",
     "cross-env": "^7.0.2",
+    "elliptic": "6.5.3",
```
Comment (stephenkr) [inline examples/demo/package.json:12]: Sorry just reviewing the dependencies, but perhaps we can remove this since it's not being used, wdyt?

**Comment by superbiche** [comment]
@stephenkr added a section to README about how to use with `runtimeConfig` and caveats for function/promise based toggles.

**Comment by stephenkr** [comment]
Hi @superbiche, sorry it took so long for me to merge, the change has been published under version 1.2.0 of the `nuxt-feature-toggle`.

Thank you again for contributing.

**Pull request merged**